### PR TITLE
scala/bug#9717 implicits in blocks to super constructor [ci: last-only]

### PR DESCRIPTION
**Pull request by Jasper-M**
Based on #5337

I'm not 100% sure yet of what I'm doing here...

### CODE REVIEW

File: test/files/pos/t9717.scala [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import scala.Predef.implicitly
```
Comment (som-snytt) [inline test/files/pos/t9717.scala:3]: PSA, you can prefer for flags
```
// scalac: -Yno-predef
```

**Comment by som-snytt** [comment]
I was also intrigued and was taking a look. I think there are related issues? that don't involve implicits?

**Comment by som-snytt** [comment]
The dotty error is fixed by another PR.

**Comment by Jasper-M** [comment]
It's always nice when it's not your own fault that things fail.

> I think there are related issues? that don't involve implicits?

I can't immediately recall any obviously related issues.

**Comment by som-snytt** [comment]
> I can't immediately recall any obviously related issues.

 Me neither. But I'll keep you updated.

**Comment by lrytz** [comment]
I spent waaaay too much time trying to understand the code in this area, but maybe there's a result.. trying it out at https://github.com/scala/scala/pull/8441.

**Comment by lrytz** [comment]
Closing in favor of https://github.com/scala/scala/pull/8441

**Pull request closed without merging**